backend/app/groups/schemas.py

```
@@ -7,6 +7,12 @@ class GroupMember(BaseModel):
     role: str = "member"  # "admin" or "member"
     joinedAt: datetime
 
+class GroupMemberWithDetails(BaseModel):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)